xlators/mount/fuse/src/fuse-bridge.h

```diff
@@ -357,7 +364,11 @@ typedef struct fuse_graph_switch_args fuse_graph_switch_args_t;
 #define fuse_log_eh_fop(this, state, frame, op_ret, op_errno)                  \
     do {                                                                       \
         fuse_private_t *priv = this->private;                                  \
-        if (this->history && priv->event_history) {                            \
+        gf_boolean_t event_history;                                            \
+        TUN_LOCK(priv);                                                        \
+        event_history = priv->event_history;                                   \
+        TUN_UNLOCK(priv);                                                      \
+        if (this->history && event_history) {                                  \
```

Comment:
I'm not sure about the rationale of that conditional, and revisiting it is out of scope for this change, so I just left its logic intact (it appears as change only because the name of the variable has changed).